package.json

```diff
@@ -82,12 +83,12 @@
     "which": "^2.0.1"
   },
   "scripts": {
-    "build-clean": "rm -rf ./packages/*/build ./packages/*/build-es5 ./packages/*/tsconfig.tsbuildinfo",
+    "build-clean": "rimraf ./packages/*/build ./packages/*/build-es5 ./packages/*/tsconfig.tsbuildinfo",
```

Comment:
`yarn` silently fails to run these scripts with `rm` not sure... It should at least fail

Reply:
cc @arcanis

Reply:
Glob patterns aren't implemented yet: https://github.com/yarnpkg/berry/issues/22

I think Rimraf does its own globbing hence why it works.

Reply:
Oh, interesting! Seems like yarn should fail if it detects globs in the scripts, then? Now it silently just doesn't work

Reply:
`rm` isn't really portable regardless, so we should probably just either use `rimraf` or write some node scripts for the `find` below

Reply:
just did that. thanks for the help!